internal/appsec/waf.go

```diff
@@ -210,6 +210,26 @@ func newHTTPWAFEventListener(handle *waf.Handle, addresses []string, timeout tim
 			values := make(map[string]interface{}, len(addresses))
 			for _, addr := range addresses {
 				switch addr {
+				case serverRequestMethodAddr:
```

Comment:
This whole chunk shouldn't be added.
You simply need to add the new switch case in the loop at line 160.